project_from_quotation/models/project_project.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+
+    quotation_id = fields.Many2one(
```

Comment:
For my comprehension and to be aligned with module name, shouldn't this exclude `sale` state orders ?